testing/MLDB-1710-left-right-rowname.py

```diff
@@ -78,7 +78,7 @@ def test_outer(self):
         res1 = mldb.query("select rightRowName() from ds1 left join ds2 on ds1.a = ds2.a + 1")
         mldb.log(res1)
 
-        expected = [["_rowName", "rightRowName()"], ["[x]-[]", '""']]
+        expected = [["_rowName", "rightRowName()"], ["[x]-[]", '']]
```

Comment:
Yes, the previous version was wrong.  This bug was fixed in this branch and the test updated.